hyperspy/misc/material.py

```diff
@@ -236,21 +237,220 @@ def density_of_mixture_of_pure_elements(weight_percent,
     Examples
     --------
     Calculate the density of modern bronze given its weight percent:
-    >>> hs.material.density_of_mixture_of_pure_elements(
-            (88, 12),("Cu", "Sn"))
+    >>> hs.material.density_of_mixture((88, 12),("Cu", "Sn"))
     8.6903187973131466
 
     """
     from hyperspy.signals import Signal
     elements = _elements_auto(weight_percent, elements)
     if isinstance(weight_percent[0], Signal):
         density = weight_percent[0]._deepcopy_with_new_data(
-            _density_of_mixture_of_pure_elements(
-                stack(weight_percent).data, elements, mean=mean))
+            _density_of_mixture(stack(weight_percent).data,
+                                elements, mean=mean))
         return density
     else:
-        return _density_of_mixture_of_pure_elements(weight_percent,
-                                                    elements, mean=mean)
+        return _density_of_mixture(weight_percent, elements, mean=mean)
+
+
+def mass_absorption_coefficient(element, energies):
+    """
+    Mass absorption coefficient (mu/rho) of a X-ray absorbed in a pure
+    material.
+
+    The mass absorption is retrieved from the database of Chantler2005
+
+    Parameters
+    ----------
+    element: str
+        The element symbol of the absorber, e.g. 'Al'.
+    energies: float or list of float or str or list of str
+        The energy or energies of the X-ray in keV, or the name of the X-rays,
+        e.g. 'Al_Ka'.
+
+    Return
+    ------
+    mass absorption coefficient(s) in cm^2/g
```

Comment:
Mmmm I think the answer is that mu isn't in barns/atom (that would be sigma) at least according to the convention followed by NIST, which is where this comes from. This is also the convention followed by Goldstein et al in "Scanning Electron Microscopy & Microanalysis" so it's the one we should follow. In any case, we don't have much choice as these are the units of the NIST database.

http://physics.nist.gov/cgi-bin/ffast/ffast.pl?Z=31&Formula=&gtype=3&lower=&upper=&density=